build.gradle.kts

```
@@ -180,12 +180,12 @@ val assembleMavenScripts = tasks.register<Zip>("assembleMavenScripts") {
     into(archiveBaseName.get())
 }
 
-tasks.named("assemble") {
-    dependsOn("assembleGradleScripts")
-    dependsOn("assembleMavenScripts")
+tasks.assemble {
+    dependsOn(assembleGradleScripts)
+    dependsOn(assembleMavenScripts)
 }
 
-tasks.register<Shellcheck>("shellcheckGradleScripts") {
+val shellcheckGradleScripts = tasks.register<Shellcheck>("shellcheckGradleScripts") {
```

You can write this like this, so you don't need to repeat the name twice:

```suggestion
val shellcheckGradleScripts by tasks.registering(Shellcheck::class) {
```